lib/did-io.js

```diff
@@ -379,6 +382,127 @@ api.promises = function(options) {
   return papi;
 };
 
+/**
+ * Register a DID document with Authorization.io.
+ * @param [options] the options to use:
```

Comment:
Add white space (with leading star) above `@param` line (separate main description with an empty line of whitespace before specifying params).